### PR TITLE
Add select table and init table

### PR DESCRIPTION
**Pull request by Semooze**
Add parameter to each functions for selecting table name when do a job.
Add init table function for using instead schema.

### CODE REVIEW

File: mod_ngarn/utils.py [hunk 1]
```diff
@@ -45,3 +53,35 @@ class ModuleNotfoundException(Exception):
...
+
+async def init_table(table: str = escape_table_name(os.getenv('DBTABLE', 'modngarn_job'))):
+    cnx = await get_connection()
+    async with cnx.transaction():
+        await cnx.execute(
+            """CREATE TABLE IF NOT EXISTS {table} (
+                    id TEXT NOT NULL CHECK (id !~ '\\|/|\u2044|\u2215|\u29f5|\u29f8|\u29f9|\ufe68|\uff0f|\uff3c'),
+                    fn_name TEXT NOT NULL,
+                    args JSONB DEFAULT '[]',
+                    kwargs JSONB DEFAULT '{}',
```
Comment (tleekkul) [inline mod_ngarn/utils.py]: Use this `kwargs JSONB DEFAULT '{{}}'`

File: mod_ngarn/worker.py [hunk 2]
```diff
@@ -31,6 +32,7 @@ class Job:
     priority: int
     args: List[Any] = field(default_factory=list)
     kwargs: Dict = field(default_factory=dict)
+    talbe: str = escape_table_name(os.getenv('DBTABLE', 'modngarn_job'))
```
Comment (tleekkul) [inline mod_ngarn/worker.py]: *table

File: mod_ngarn/worker.py [hunk 3]
```diff
@@ -54,7 +56,7 @@ class Job:
     async def success(self, result: Dict, processing_time: Decimal) -> str:
         """ Success execution handler """
         return await self.cnx.execute(
-            "UPDATE modngarn_job SET result=$1, executed=NOW(), processed_time=$2 WHERE id=$3",
+            f"UPDATE {self.talbe} SET result=$1, executed=NOW(), processed_time=$2 WHERE id=$3",
```
Comment (tleekkul) [inline mod_ngarn/worker.py]: *self.table

File: mod_ngarn/worker.py [hunk 4]
```diff
@@ -68,7 +70,7 @@ class Job:
             'Rescheduled, delay for {} seconds ({}) '.format(delay, next_schedule.isoformat())
         )
         return await self.cnx.execute(
-            "UPDATE modngarn_job SET priority=priority+1, reason=$2, scheduled=$3  WHERE id=$1",
+            f"UPDATE {self.talbe} SET priority=priority+1, reason=$2, scheduled=$3  WHERE id=$1",
```
Comment (tleekkul) [inline mod_ngarn/worker.py]: *self.table

File: mod_ngarn/worker.py [hunk 5]
```diff
@@ -77,18 +79,23 @@ class Job:
...
+        self,
+        cnx: asyncpg.Connection,
+        table: str = escape_table_name(os.getenv('DBTABLE', 'modngarn_job')),
+    ):
+
         result = await cnx.fetchrow(
-            """
-            SELECT id, fn_name, args, kwargs, priority
-            FROM modngarn_job 
+            """SELECT id, fn_name, args, kwargs, priority FROM {table}
```
Comment (tleekkul) [inline mod_ngarn/worker.py]: We cannot use $1 here right?

Comment (Semooze) [reply mod_ngarn/worker.py]: yes

File: mod_ngarn/utils.py (hunk 1 above)
Comment (KayEss) [reply mod_ngarn/utils.py]: Why `JSONB` and not `JSON` (or even `text`) for these columns?

File: mod_ngarn/worker.py (hunk 5 above)
Comment (KayEss) [reply mod_ngarn/worker.py]: Probably you want to put the table name in double quotes though.

**Pull request closed without merging**